Fixes big and little endian errors

johnnyman727 opened the pull request:
* Fixes sign errors by using the proper endianness for the relevant variables
* Picks a more consistent package name
* Adds me to the authors

Still need to have an API exposed and published to NPM. We could also add a pressure API (since it is a barometer)

sarmadsangi commented:
@johnnyman727 thanks for fixing sign errors, i ll expose it as a function for temperature reading in next PR. Will merge this but seems like there is a minor conflict in `package.json`.

johnnyman727 commented:
Updated! I reverted by changes to the package.json, but I'd recommend you change the name to "baro-bmp180" to be consistent with the "function-chipId" naming scheme the community has been using for a while.

![screen shot 2016-11-26 at 6 19 18 pm](https://cloud.githubusercontent.com/assets/432909/20639946/ac99abd8-b40c-11e6-9798-e3441339c5f8.png)

sarmadsangi commented:
@johnnyman727 awesome, thanks will do it post merge.

sarmadsangi commented:
Hey @johnnyman727 i have updated `index.js` added a callback and a example in `test.js`. Since i don't have sensor with at this moment it would be difficult for me to test. 

If you happen to have sensor with you, can you execute `test.js` and see if all good (before i publish this to npm)
https://github.com/sarmadsangi/tessel-baro-bmp180/blob/master/test.js

johnnyman727 commented:
Hey @sarmadsangi I'll give this a go tomorrow. Thanks for working on it!
